compass/ocean/tests/sphere_transport/correlated_tracers_2d/mesh.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+
+from mpas_tools.ocean import build_spherical_mesh
+
+from compass.step import Step
+from compass.model import make_graph_file
+
+
+class Mesh(Step):
+    """
+    A step for creating uniform global meshes
+
+    Attributes
+    ----------
+    resolution : int
+        The resolution of the (uniform) mesh in km
+    """
+
+    def __init__(self, test_case, resolution):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case :
+            compass.ocean.tests.sphere_transport.correlatedTracers2D.CorrelatedTracers2D
```

Comment:
Same comment about line length -- this needs to go back to being one line.
And again, the description got chopped in the last commit